### DISCUSSION TITLE
Is it possible to indicate in grey Everything search results that are currently not available ?

### DISCUSSION THREAD

**Discussion by oeloo** [Flow-Launcher/Flow.Launcher#2029] [General]
Hi, Everything Search v1.5a displays results that are currently not available (example, because you have not currently your external hard-drive plugged) in gray color instead of white :
![image](https://user-images.githubusercontent.com/98819304/229815383-cacbe926-792f-416f-9aba-f95c895e56c1.png)
This gives a good indication about the result while telling the user it is not currently available.

When I search the same thing in Flow Launcher, I have the same result but currently unavailable files are not in grey color :
![image](https://user-images.githubusercontent.com/98819304/229818502-fb9604da-df86-49ab-af44-4dd4a4ac2500.png)
Is there a possibility to have them in grey color in Flow Launcher ? Regads

**Comment by taooceros** [comment]
We are still using the 1.4 api. I don't think that's possible for now (before they release the 1.5 api. Have they?).

It would be a nice feature to add though but likely solution will be to omit those unavailable result.

**Comment by oeloo** [comment]
So this would a future request for API 1.5 :). Everything v1.5 is already pretty stable, though it is officially alpha version. A lot user have already started to use v1.5, instead of v1.4.

Hiding these results would not be a good decision and Flow Launcher is already showing them I have seen : so you have nothing to modify.
But displaying them with in gray like in Everything v1.5 would be necessary for the end user: so he is not surprised nothing comes up if he click on it. He first needs to plug his external hard-drive for this.
Concretely this means the last item (with path F:\) in my 2nd screenshot should be gray.
